kythe/java/com/google/devtools/kythe/extractors/java/standalone/javac-wrapper.sh

```diff
@@ -40,6 +40,9 @@
 
 export TMPDIR="${TMPDIR:-/tmp}"
 
+if [[ -z "$JAVABIN" ]]; then
+  readonly JAVABIN="$(which java)"
```

Comment:
huh wonder if we need REAL_JAVAC still then.

maybe fine to leave in

Reply:
I changed it to use `JAVA_HOME` instead.

The reason `javac` uses a different var because this wrapper be be added to `PATH` to replace `javac` silently.